apps/admin/src/components/customFields/metadataLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Buildable, WrappedInput, Field } from '@daohaus/ui';
+
+const presetLabels: { [key: string]: boolean } = {
+  github: true,
+  web: true,
+  discord: true,
+  twitter: true,
+  telegram: true,
+  blog: true,
+};
+
+export const MetadataLink = (props: Buildable<Field>) => {
```

Comment:
Why is the component file name camel case?
Looks to be a pretty normal react component

Reply:
Good catch - i'll clean up a few more of thse custom inputs with camel case name too.